fix typo (#2749)
huggingface/blog: nvidia-physical-ai.md

---
title: "NVIDIA's GTC 2025 Announcement for Physical AI Developers: New Open Models and Datasets" 

> thumbnail: /blog/assets/101_decision-transformers-train/thumbnail.gif

Reviewer: ```suggestion
thumbnail: /blog/assets/nvidia-physical-ai/thumbnail.png
```

authors:
- user: mingyuliutw
  guest: true
  org: nvidia
- user: hannamao
  guest: true
  org: nvidia
- user: jwgu
  guest: true
  org: nvidia
- user: PranjaliJoshi
  guest: true
  org: nvidia
- user: asawareeb
  guest: true

>   org: nvidia

Reviewer: This user does not belong to the nvidia org in the Hub.

---

| ![Humanoid pick and place](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/blog/physical-ai/humanoid.gif) |
| :--: |
| *NVIDIA Isaac GR00T N1 used in object manipulation.* |

> At its annual GTC conference, NVIDIA has unveiled a trio of groundbreaking open-source releases aimed at accelerating physical AI development. Release of a new suite of world foundation models(WFMs) with multicontrols called **Cosmos Transfer**, a highly curated **Physical AI Dataset**, and the first open model for general humanoid reasoning called **NVIDIA Isaac GR00T N1** - represent a significant leap forward in physical AI technology, offering developers powerful tools and resources to advance robotics systems, and enhance autonomous vehicle technology.

Reviewer: ```suggestion
At its annual GTC conference, NVIDIA has unveiled a trio of groundbreaking open-source releases aimed at accelerating physical AI development. Release of a new suite of world foundation models (WFMs) with multicontrols called **Cosmos Transfer**, a highly curated **Physical AI Dataset**, and the first open model for general humanoid reasoning called **NVIDIA Isaac GR00T N1** - represent a significant leap forward in physical AI technology, offering developers powerful tools and resources to advance robotics systems, and enhance autonomous vehicle technology.
```

> # New World Foundation Model - Cosmos Transfer

Reviewer: ```suggestion
## New World Foundation Model - Cosmos Transfer
```

[Cosmos Transfer](https://huggingface.co/collections/nvidia/cosmos-transfer1-67c9d328196453be6e568d3e), the latest addition to NVIDIA's Cosmos™ world foundation models (WFMs), introduces a new level of control and accuracy in generating virtual world scenes.

Available in 7 billion parameter size, the model utilizes multicontrols to guide the generation of high-fidelity world scenes from structural inputs, ensuring precise spatial alignment and scene composition.

> ## How it works

Reviewer: ```suggestion
### How it works
```

| ![Cosmos Transfer output](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/blog/physical-ai/output_1.gif) |
| :--: |
| *Outputs from Cosmos Transfer varying environments and weather conditions.* |

| ![Cosmos Transfer output](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/blog/physical-ai/output_2.gif) |
| :--: |
| *Outputs from Cosmos Transfer varying environments and weather conditions.* |

| ![Cosmos Transfer output](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/blog/physical-ai/output_3.gif) |
| :--: |
| *Outputs from Cosmos Transfer varying environments and weather conditions.* |

Cosmos Transfer coupled with the NVIDIA Omniverse platform is driving controllable synthetic data generation for robotics and autonomous vehicle development at scale. Find more Cosmos Transfer [Examples](https://github.com/nvidia-cosmos/cosmos-transfer1#examples) on GitHub.

Cosmos Transfer [samples](https://huggingface.co/nvidia/Cosmos-Transfer1-7B-Sample-AV) built using post-training base model are also available for autonomous vehicles. 

> # Open Physical AI Dataset

Reviewer: ```suggestion
## Open Physical AI Dataset
```

NVIDIA has also released **Physical AI Dataset**, an open-source dataset on [Hugging Face](https://huggingface.co/collections/nvidia/physical-ai-67c643edbb024053dcbcd6d8) for developing physical AI. This commercial-grade, pre-validated dataset consists of 15 terabytes of data representing more than 320,000 trajectories for robotics training, plus up to 1,000 Universal Scene Description (OpenUSD) assets, including a SimReady collection.

The dataset is designed for post-training foundation models like Cosmos Predict world foundation models, providing developers with high-quality, diverse data to enhance their AI models.

> # Purpose Built Model for Humanoids - NVIDIA Isaac GR00T N1

Reviewer: ```suggestion
## Purpose Built Model for Humanoids - NVIDIA Isaac GR00T N1
```

> Another exciting announcement is the release of [NVIDIA Isaac GR00T N1](https://developer.nvidia.com/blog/accelerate-generalist-humanoid-robot-development-with-nvidia-isaac-gr00t-n1/), the world's first open foundation model for generalized humanoid robot reasoning and skills. This cross-embodiment model takes multimodal input, including language and images, to perform manipulation tasks in diverse environments. The NVIDIA Isaac GR00T-N1-2B model is available on [Hugging Face](https://huggingface.co/nvidia/GR00T-N1-2B).

Author: ```suggestion
Another exciting announcement is the release of [NVIDIA Isaac GR00T N1](https://developer.nvidia.com/blog/accelerate-generalist-humanoid-robot-development-with-nvidia-isaac-gr00t-n1/), the world's first open foundation model for generalized humanoid robot reasoning and skills. This cross-embodiment model takes multimodal input, including language and images, to perform manipulation tasks in diverse environments. The NVIDIA Isaac GR00T-N1-2B model is available on [Hugging Face](https://huggingface.co/nvidia/GR00T-N1-2B).
```

> Another exciting announcement is the release of [NVIDIA Isaac GR00T N1](https://developer.nvidia.com/blog/accelerate-generalist-humanoid-robot-development-with-nvidia-isaac-gr00t-n1/), the world's first open foundation model for generalized humanoid robot reasoning and skills. This cross-embodiment model takes multimodal input, including language and images, to perform manipulation tasks in diverse environments. The NVIDIA Isaac GR00T-N1-2B model is available on [Hugging Face](https://huggingface.co/nvidia/Isaac-GR00T-N1-2B).

Reviewer: ```suggestion
Another exciting announcement is the release of [NVIDIA Isaac GR00T N1](https://developer.nvidia.com/blog/accelerate-generalist-humanoid-robot-development-with-nvidia-isaac-gr00t-n1/), the world's first open foundation model for generalized humanoid robot reasoning and skills. This cross-embodiment model takes multimodal input, including language and images, to perform manipulation tasks in diverse environments. The NVIDIA Isaac GR00T-N1-2B model is available on [Hugging Face](https://huggingface.co/nvidia/GR00T-N1-2B).
```

Isaac GR00T N1 features a dual-system architecture inspired by human cognition, consisting of the following complementary components:

- **Vision-Language Model (System 2)**: This methodical thinking system is based on [NVIDIA-Eagle](https://huggingface.co/NVEagle) with [SmolLM-1.7B](https://huggingface.co/HuggingFaceTB/SmolLM-1.7B). It interprets the environment through vision and language instructions, enabling robots to reason about their environment and instructions, and plan the right actions.
- **Diffusion Transformer (System 1)**: This action model generates continuous actions to control the robot's movements, translating the action plan made by System 2 into precise, continuous robot movements.

> # Path Forward

Reviewer: ```suggestion
## Path Forward
```

Post-training is the path forward to advancing autonomous systems, creating specialized models for downstream physical AI tasks.

> Check out GitHub for [Cosmos Predict](https://github.com/nvidia-cosmos/cosmos-predict1) and [Cosmos Transfer](https://github.com/nvidia-cosmos/cosmos-transfer1) inference scripts. Explore the Cosmos Transfer [research paper](https://research.nvidia.com/publication/2025-03_cosmos-transfer-1-world-world-transfer-adaptive-multi-control-physical-ai) for more details.

Author: ```suggestion
Check out GitHub for [Cosmos Predict](https://github.com/nvidia-cosmos/cosmos-predict1) and [Cosmos Transfer](https://github.com/nvidia-cosmos/cosmos-transfer1) inference scripts. Explore the Cosmos Transfer [research paper](https://research.nvidia.com/labs/dir/cosmos-transfer1/) for more details.
```

> The NVIDIA Isaac GR00T-N1-2B model is available on [Hugging Face](https://huggingface.co/nvidia/Isaac-GR00T-N1-2B). Sample datasets and PyTorch scripts for post-training using custom user datasets, which is compatible with the Hugging Face LeRobot format are available on [GitHub](http://github.com/NVIDIA/Isaac-GR00T). For more information about the Isaac GR00T N1 model, see the [research paper](https://research.nvidia.com/publication/2025-03_nvidia-isaac-gr00t-n1-open-foundation-model-humanoid-robots).

Reviewer: ```suggestion
The NVIDIA Isaac GR00T-N1-2B model is available on [Hugging Face](https://huggingface.co/nvidia/GR00T-N1-2B). Sample datasets and PyTorch scripts for post-training using custom user datasets, which is compatible with the Hugging Face LeRobot format are available on [GitHub](http://github.com/NVIDIA/Isaac-GR00T). For more information about the Isaac GR00T N1 model, see the [research paper](https://research.nvidia.com/publication/2025-03_nvidia-isaac-gr00t-n1-open-foundation-model-humanoid-robots).
```
